allensdk/brain_observatory/behavior/data_files/stimulus_file.py

```diff
@@ -29,8 +28,8 @@
 """
 
 
-def from_json_cache_key(cls, dict_repr: dict):
-    return hashkey(json.dumps(dict_repr))
+def from_json_cache_key(cls, stimulus_file_path: str):
```

Comment:
For my own reference: is this something we want to do in general with these (thinking of the case of the syncfile)? As in not use the full dict but only the filepath?

Reply:
I changed this from `dict_repr` due to the way that these stimulus files are created. We now have several subclasses (`MappingStimulusFile`, `ReplayStimulusFile`, etc) that all inherit the same `from_json`. They pass the same `dict_repr` every time. So 1 object is cached and returned every time which wasn't right here. 

As to syncfile, I'm not sure. We just need to make sure the key being cached is unique for different requests.